### PR TITLE
Add more typos for "configure" and similar

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12167,10 +12168,21 @@ condidtioning->conditioning
 condidtions->conditions
 condifurable->configurable
 condifuration->configuration
+condifurations->configurations
 condifure->configure
 condifured->configured
+condifures->configures
+condifuring->configuring
 condig->config
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:12176]: Perhaps
```suggestion
condig->config, condign,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12318,33 +12350,65 @@ confidantially->confidentially
 confidental->confidential
 confidentally->confidentially
 confids->confides
+confif->config
+confifs->configs
 confifurable->configurable
 confifuration->configuration
+confifurations->configurations
 confifure->configure
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:12358]:
```suggestion
confifure->configure, confiture,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -12318,33 +12350,65 @@ confidantially->confidentially
...
 confifured->configured
+confifures->configures
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:12360]:
```suggestion
confifures->configures, confitures,
```

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:12176]: I will add `coding` and `condign`.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**